concourse/pipelines/rhui-release.jsonnet

```diff
@@ -0,0 +1,142 @@
+local project = 'google.com:rhel-infra';
+
+local gatejob = {
+  local job = self,
+
+  name: error 'must set name on gatejob',
+  trigger:: true,
+  passed:: [],
+  plan: [
+    {
+      get: imageResource,
+      trigger: job.trigger,
+      passed: job.passed,
+    }
+    for imageResource in ['cds-image', 'rhua-image']
+  ],
+};
+
+local gcloudmigtask = {
+  local task = self,
+
+  node:: error 'must set node on gcloudmigtask',
+  stage:: 'prod',
+  region:: error 'must set region on gcloudmigtask',
+  action:: error 'must set action on gcloudmigtask',
+
+  task: error 'must set task',
```

Comment:
nit: must set task in gcloudimgtask

Reply:
done

Reply:
it says done but.. it is not done? (i tried refreshing, lmk if it's on my end)

Reply:
whoops, sorry, I have it staged locally. Pushed.